openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1119,6 +1121,91 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used to receive notification from the Wallet whether credential has been successfully received in the Wallet or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL.
```

Comment:
Can we rewrite the first sentence as:

"This endpoint is used to receive notifications from the Wallet indicating whether a credential has been successfully received or not."

just to make it a little cleaner

Reply:
done